app/views/devise/passwords/edit.html.erb

```diff
@@ -5,13 +5,16 @@
   <%= f.hidden_field :reset_password_token %>
 
   <div class="field">
-    <%= f.label :password, "New password" %><br />
+    <%= f.label :password, "New password" %> <br />
     <% if @minimum_password_length %>
       <em>(<%= @minimum_password_length %> characters minimum)</em><br />
     <% end %>
     <%= f.password_field :password, autofocus: true, autocomplete: "off" %>
-  </div>
 
+  </div>
+  <br /> 
```

Comment:
`<br>` shouldn't be used for spacing. Use it for a new line in a sentence, poem, etc. Margins would be better here.

Reply:
Ok